### PR TITLE
Changes for Community Badges

### PR DESCRIPTION
**Pull request by ifeomaro**

### CODE REVIEW

**Review by aliillyas — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -4,6 +4,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.1.0'
+        classpath 'com.layer:gradle-git-repo-plugin:2.0.2'
```
Comment (ifeomaro) [inline build.gradle]: Since we now use the S3 bucket, this is no longer needed.

File: lib/build.gradle [hunk 2]
```diff
@@ -1,21 +1,46 @@
 apply plugin: 'com.android.library'
 apply plugin: 'maven'
-apply plugin: 'signing'
-//apply from: '../.publishing/sonatype.gradle'
+apply plugin: 'git-repo'
```
Comment (ifeomaro) [inline lib/build.gradle]: Not needed

File: lib/build.gradle [hunk 3]
```diff
@@ -28,3 +53,106 @@ dependencies {
...
+        maven {
+            url "file://${gitPublishConfig.home}/${gitPublishConfig.org}/${gitPublishConfig.repo}/releases"
+        }
+    }
+
+    publish.dependsOn assembleRelease
+
+}
+
+gitPublishConfig {
```
Comment (ifeomaro) [inline lib/build.gradle]: Not needed

File: lib/build.gradle [hunk 4]
```diff
@@ -28,3 +53,106 @@ dependencies {
...
+    org = 'WW-Digital'
+    repo = 'android-gradle-repo'
+}
+
+repositories {
+    flatDir {
+        dirs 'aar'
+    }
+
+    github("${gitPublishConfig.org}", "${gitPublishConfig.repo}")
```
Comment (ifeomaro) [inline lib/build.gradle]: Not needed

**Review by ifeomaro — COMMENTED** [review]
I added a few comments. We need to update this to use the S3 bucket setup. Other than that, this looks good.

**Review by aliillyas — APPROVED** [review]

**Comment by aliillyas** [comment]
looks good

**Pull request merged**